source/guides/network/running-a-test-net-node.rst

```diff
@@ -9,154 +9,196 @@
+            vrf:
+                privateKey: ●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●●
+                publicKey: 1BB864D80F9FC8BF661265276E00E57C1ABDD13B1454A3F5ADC025279FA03268
+                address: TD74YTRSVU4HXEXE4LQYIX2EQ65XLYO4V5XQB4I
+
+.. note:: Keep you Secret Keys secret at all times!
+
+Use the information in the ``signing`` section to access the node's account.
+
+***************************
+Providing funds to the node
+***************************
+
+Before the node can begin harvesting or voting (see below) it needs a **minimum amount of funds**. In the ``testnet`` test environment you can use |codename|'s **Faucet** to provide these funds.
+
+Go to |faucet-1| (or |faucet-2|), insert the ``address`` of your node's account and the amount of |networkcurrency| you need and click on CLAIM.
 
-    sudo docker-compose up --build --detach
+You will see that your request is first ``unconfirmed`` (pending) and after a few seconds it becomes ``confirmed``. Your node is now funded and ready to operate!
 
-You should see docker downloading the container images for the first time. Then it should run the setup and finally startup the service.
+********************
+Submitting link keys
+********************
 
-To stop all the running services, run ``sudo docker-compose down`` in the same directory you executed the ``up`` command.
+**Harvesting** and **Voting** nodes require an extra configuration step before they can be used:
 
-Peer assembly
-=============
+Enabling harvesting
+===================
 
-You can verify that the node is running by running with the command ``docker-compose ps peer-node`` in the same ``peer-assembly`` folder.
-The command's output looks like:
+|symbol-bootstrap| creates peer nodes with :doc:`harvesting <../../concepts/harvesting>` enabled by default, but they still need to be registered by announcing a :ref:`VrfKeyLinkTransaction <vrf-key-link-transaction>` to the network.
+
+This can be done by |symbol-bootstrap| too, but it needs to be a step separated from ``symbol-bootstrap start`` because funds are required to announce transactions.
+
+Once the node is running with ``symbol-bootstrap start`` and you have funded it account, from a different terminal (but from the same folder), simply type:
 
 .. code-block:: bash
 
-     Name                       Command                    State   Ports
-     ------------------------------------------------------------------------------------
-     peerassembly_peer-node_1   bash -c /bin-mount/wait    Up      0.0.0.0:7900->7900/tcp
+    symbol-bootstrap link
 
-The node is running if the state for ``peerassembly_peer-node_1`` is set to "Up".
+This creates the required :ref:`VrfKeyLinkTransaction <vrf-key-link-transaction>` and submits it to the network. If it succeeds (it might take some seconds, as the transaction needs to be approved and incorporated into the blockchain) your new node is ready to harvest.
```

Comment:
not sure it is good to use `incorporated` here? 
Shall we say `Once the transaction gets confirmed` instead?

Reply:
I would prefer `confirmed` too.

Reply:
Hmmm, yeah, poor choice of words on my side :)